docs/changelog.md

```diff
@@ -1380,7 +1380,7 @@ v0.23
       `git_submodule_set_fetch_recurse_submodules()`.
 
 * `git_submodule_status()` no longer takes a submodule instance but a
-  repsitory, a submodule name and an ignore setting.
+  repository, a submodule name and an ignore setting.
```

Comment:
I regret that we made spelling errors _but_ I don't love changing the `changelog` after the fact.  It feels like once we've released the release that we should keep the corresponding changelog static (warts and all).